src/pybind/mgr/cephadm/templates/services/ingress/haproxy.cfg.j2

```diff
@@ -73,17 +76,24 @@ backend backend
 {% endif %}
     balance static-rr
     option httpchk HEAD / HTTP/1.0
-    {% for server in servers %}
-    server {{ server.name }} {{ server.ip }}:{{ server.port }} check weight 100 inter {{ health_check_interval }}
-    {% endfor %}
 {% endif %}
 {% if mode == 'tcp' %}
     mode        tcp
-    balance     source
-    hash-type   consistent
```

Comment:
just out of curiosity, why are we removing these two options? .. if there's a good reason I think we should document it in the PR explicitly just to keep a record of the change.